### DISCUSSION TITLE
No data in LOOKUP column [Dec 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4710] [Discord Threads]
> 🙋‍♂️ **baiyu**  
## Hi, I try to use LOOKUP columns, but when I create it, these is without any data。I already have two sheet  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1044092259081142333/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### in total_project, do you have a LTAR (LinkToAnotherRecord) column?  
#  
> 🙋‍♂️ **baiyu**  
#### Thank you !!! I don't create the LTAR column in total_project. When I create it, I got the data in lookup columns.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1044092259387318292)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
